gcp-common/src/main/java/io/micronaut/discovery/cloud/gcp/GoogleComputeInstanceMetadataResolver.java

```diff
@@ -15,15 +15,15 @@
  */
 package io.micronaut.discovery.cloud.gcp;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import io.micronaut.context.annotation.Primary;
 import io.micronaut.context.annotation.Requires;
 import io.micronaut.context.env.Environment;
 import io.micronaut.core.util.StringUtils;
 import io.micronaut.discovery.cloud.ComputeInstanceMetadata;
 import io.micronaut.discovery.cloud.ComputeInstanceMetadataResolver;
 import io.micronaut.discovery.cloud.NetworkInterface;
+import io.micronaut.json.tree.JsonNode;
+import io.micronaut.serde.ObjectMapper;
```

Comment:
think you can support both Jackson and Micronaut serialisation if you use `JsonMapper.createDefault()` instead

Reply:
See https://github.com/micronaut-projects/micronaut-oracle-cloud/blob/510daa6a79bf2c25843e652bd7140260d85b5e92/oraclecloud-common/src/main/java/io/micronaut/discovery/cloud/oraclecloud/OracleCloudMetadataResolver.java